Realm/RLMRealmConfiguration.h

```diff
@@ -71,6 +71,9 @@ RLM_ASSUME_NONNULL_BEGIN
 /// The block which migrates the Realm to the current version.
 @property (nonatomic, copy, nullable) RLMMigrationBlock migrationBlock;
 
+/// Delete the Realm file if migration is needed and re-create with the new schema.
+@property (nonatomic, getter = shouldDeleteRealmIfMigrationNeeded) BOOL deleteRealmIfMigrationNeeded;
```

Comment:
the custom getter here seems superfluous. `deleteRealmIfMigrationNeeded` reads like a fine getter name to me.